sdk/identity/identity/samples-dev/environmentCredential.ts

```diff
@@ -1,6 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * @summary Authenticates with a client and a client's secret sent through environment variables.
```

Comment:
```suggestion
 * @summary Authenticates as an app registration automatically using environment variables.
```

The wording is a little awkward.